testing/e2e/skr-tester/cmd/main.go

```diff
@@ -0,0 +1,29 @@
+package main
+
+import (
+	"fmt"
+	"os"
+	"os/signal"
+	"skr-tester/pkg/command"
+	"syscall"
+)
+
+func main() {
+	setupCloseHandler()
+	cmd := command.New()
+
+	err := cmd.Execute()
+	if err != nil {
```

Comment:
As for YAGNI principle, we could leave this `1` assuming that no elaborate shell scripts will call this CLI tool.

Reply:
I don't foresee a complex script using this tool. The script will only execute commands from this tool.